### PR TITLE
04_0_Enviando_Transacciones_Bitcoin.md

### PR DESCRIPTION
**Pull request by srahalh**

### CODE REVIEW

**Review by javiervargas — COMMENTED** [review]
pending fix links

**Comment by icculp** [comment]
Need to rename the file to 04_0_Enviando_Transacciones_Bitcoin.md, appending ".md" to the filename, otherwise github won't recognize it as markdown and the formatting doesn't show.

File: es/04_0_Enviando_Transacciones_Bitcoin [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+   * Entender las transacciones y sus comisiones
+   * Entender las transacciones de tipo Legacy & SegWit
+   * Usar métodos básicos para enviar dinero 
+   * Usar métodos automaticos para calcular comisiones para enviar dinero
+   * Entender el peligro de las transacciones sin procesar
+   
+## Tabla de contenido
+  
+  * [Sección Uno: Enviando Monedas de Forma Sencilla](04_1_Sending_Coins_The_Easy_Way.md)
```
Comment (icculp) [inline es/04_0_Enviando_Transacciones_Bitcoin:23]: `* [Sección Uno: Enviando Monedas de la Forma Fácil](04_0_Enviando_Transacciones_Bitcoin.md)`
to match the titles and links in LEEME.md

File: es/04_0_Enviando_Transacciones_Bitcoin [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  * [Sección Dos: Creando una Transacción en Crudo](04_2_Creating_a_Raw_Transaction.md)
```
Comment (icculp) [inline es/04_0_Enviando_Transacciones_Bitcoin:24]: `[4.2: Creando una Transacción Cruda](04_2_Creando_una_Transaccion_Cruda.md)`

File: es/04_0_Enviando_Transacciones_Bitcoin [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+     * [Interludio: Usando JQ](04_2__Interlude_Using_JQ.md)
```
Comment (icculp) [inline es/04_0_Enviando_Transacciones_Bitcoin:25]: `[Interludio: Usando JQ](04_2__Interludio_Usando_JQ.md)`

File: es/04_0_Enviando_Transacciones_Bitcoin [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  * [Sección Tres: Creando Transacciones en Crudo Usando Argumentos con Nombre](04_3_Creating_a_Raw_Transaction_with_Named_Arguments.md)
```
Comment (icculp) [inline es/04_0_Enviando_Transacciones_Bitcoin:26]: `[4.3: Creando una Transacción Cruda con Argumentos Ingresados con Nombre](04_3_Creando_una_Transaccion_Cruda_con_Argumentos_Ingresados_con_Nombre.md)`

File: es/04_0_Enviando_Transacciones_Bitcoin [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  * [Sección Cuatro: Enviando Monedas con Transacciones en Crudo](04_4_Sending_Coins_with_a_Raw_Transaction.md)
```
Comment (icculp) [inline es/04_0_Enviando_Transacciones_Bitcoin:27]: `[4.4: Enviando Monedas con una Transacción Cruda](04_4_Enviando_Monedas_con_una_Transaccion_Cruda.md)`

**Comment by javiervargas** [comment]
x

> Need to rename the file to 04_0_Enviando_Transacciones_Bitcoin.md, appending ".md" to the filename, otherwise github won't recognize it as markdown and the formatting doesn't show.

Sure,  I think it happens on all four chapters.

File: es/04_0_Enviando_Transacciones_Bitcoin (hunk 5 above)
Comment (icculp) [reply es/04_0_Enviando_Transacciones_Bitcoin:27]: `     * [Interludio: Usando Curl](04_4__Interludio_Usando_Curl.md)`
add interlude, PR #427

File: es/04_0_Enviando_Transacciones_Bitcoin [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  * [Sección Cinco: Enviando Monedas con Transacciones Automatizadas en Crudo](04_5_Sending_Coins_with_Automated_Raw_Transactions.md)
```
Comment (icculp) [inline es/04_0_Enviando_Transacciones_Bitcoin:28]: `[4.5: Enviando Monedas con Transacciones Crudas Automatizadas](04_5_Enviando_Monedas_con_Transacciones_Crudas_Automatizadas.md)`

File: es/04_0_Enviando_Transacciones_Bitcoin [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  * [Sección Seis: Creando Transacciones SegWit](04_6_Creating_a_Segwit_Transaction.md)
```
Comment (icculp) [inline es/04_0_Enviando_Transacciones_Bitcoin:29]: `[4.6: Creando una Transacción Segwit](04_6_Creando_una_Transaccion_Segwit.md)`

**Review by icculp — CHANGES_REQUESTED** [review]

**Review by icculp — APPROVED** [review]

**Comment by icculp** [comment]
I think it looks good, @shannona ready to be merged

**Pull request merged**